docs/resources/uberbank_example1.ini

```diff
@@ -97,19 +102,19 @@ min-match = 0.97
 random-seed = 1915
 pn-order = threePointFivePN
 f-low = 30
-f-upper = 1100
+f-upper = 1000
 delta-f = 0.01
-min-mass1 = 1.
-max-mass1 = 48.
-min-mass2 = 1.
-max-mass2 = 48.
-min-total-mass = 2.
-max-total-mass = 6.
-max-bh-spin-mag = 0.9899
-max-ns-spin-mag = 0.05
-ns-bh-boundary-mass = 2.
-asd-file = /home/spxiwh/paper_work/bbh-aligned-spin-paper/asd-T1200307v4.txt
-split-bank-num = 25
+min-mass1 = 1
+max-mass1 = 4
+min-mass2 = 1
+max-mass2 = 4
+min-total-mass = 2
+max-chirp-mass = 1.5
+max-bh-spin-mag = ${workflow|max-bh-spin}
+max-ns-spin-mag = ${workflow|max-ns-spin}
+ns-bh-boundary-mass = ${workflow|nsbh-boundary}
```

Comment:
Maybe stick the two PSD file inputs at the top as well, so that they live together. Then if a user changes one they will also clearly see that the other should be changed as well ... But we should probably persuade the codes to take the same format input one way or the other (or both)